### ISSUE TITLE
Overlay won't run on updated ACT. 

### ISSUE THREAD

**Issue by Marscgm** [hibiyasleep/kagerou#134]
Title. Files unblocked. Folder is not read only.  Firewall unblocked/tried with it off. Tired uninstalling/reinstalling/uninstalling/reinstalling. Program is always ran as admin. ect.

Edit: Worked through the exception. New issue:

Plugin Initialization Failed

Could not load file or assembly 'Xilium.CefGlue, Version=3.2272.2035.0c Culture=neutral, PublicKeyToken=6235298024de30d5' or one of it's dependencies. The system cannot find the file specified.

**Comment by Marscgm** [comment]
Actually. It's no longer working in the 86 client.. I think the above message was from that. This should be the 64 client.

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at RainbowMage.OverlayPlugin.OverlayBase`1.<InitializeConfigHandlers>b__12(Object o, VisibleStateChangedEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at RainbowMage.OverlayPlugin.OverlayConfigBase.set_IsVisible(Boolean value)
   at System.Windows.Forms.CheckBox.OnCheckedChanged(EventArgs e)
   at System.Windows.Forms.CheckBox.set_CheckState(CheckState value)
   at System.Windows.Forms.CheckBox.OnClick(EventArgs e)
   at System.Windows.Forms.CheckBox.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3801.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
Advanced Combat Tracker
    Assembly Version: 3.4.0.260
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3801.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3815.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
GammaJul.LgLcd
    Assembly Version: 1.1.3.0
    Win32 Version: 1.1.3.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/GammaJul.LgLcd.DLL
----------------------------------------
ICSharpCode.SharpZipLib
    Assembly Version: 0.85.5.452
    Win32 Version: 0.85.5.452
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/ICSharpCode.SharpZipLib.DLL
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Runtime.Serialization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3825.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll
----------------------------------------
SMDiagnostics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3825.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/SMDiagnostics/v4.0_4.0.0.0__b77a5c561934e089/SMDiagnostics.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
FFXIV_ACT_Plugin
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Memory
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Common
    Assembly Version: 2.0.2.2
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Config
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.LogFile
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Network
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Overlay
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Parse
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
FFXIV_ACT_Plugin.Resource
    Assembly Version: 2.0.2.7
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
Machina
    Assembly Version: 2.1.1.0
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
Machina.FFXIV
    Assembly Version: 2.1.3.2
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 4.8.3801.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
OverlayPlugin
    Assembly Version: 0.3.4.0
    Win32 Version: 3.4.0.260
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Advanced%20Combat%20Tracker.exe
----------------------------------------
OverlayPlugin.Common
    Assembly Version: 0.3.4.0
    Win32 Version: 0.3.4.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Overlay/OverlayPlugin.Common.dll
----------------------------------------
OverlayPlugin.Core
    Assembly Version: 0.3.4.0
    Win32 Version: 0.3.4.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Overlay/OverlayPlugin.Core.dll
----------------------------------------
HtmlRenderer
    Assembly Version: 1.0.1.8
    Win32 Version: 1.0.1.8
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Overlay/HtmlRenderer.dll
----------------------------------------
Xilium.CefGlue
    Assembly Version: 3.2272.2035.0
    Win32 Version: 3.2272.2035.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/Overlay/Xilium.CefGlue.dll
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 4.8.3761.0 built by: NET48REL1

**Comment by Marscgm** [comment]
![image](https://user-images.githubusercontent.com/10990130/61576957-23947f80-aa9e-11e9-8a40-fbd919885be8.png)

**Comment by Marscgm** [comment]
Got it to work via using an out of date ACT (roughly 35 days old), not using the set up wizard, installing and starting the overlay, then updating ACTthen using the start up wizard.

Would like to keep this open as not everyone has an older version of ACT.
